packages/fetch/src/index.ts

```diff
@@ -114,10 +114,8 @@ ${
     operationName,
   );
 
-  const responseDataType =
-    response.definition.success || response.definition.errors
-      ? `${response.definition.success !== 'unknown' ? response.definition.success : ''}${response.definition.success !== 'unknown' && response.definition.errors !== 'unknown' ? ' | ' : ''}${response.definition.errors !== 'unknown' ? response.definition.errors : ''}`
-      : 'unknown';
+  let responseDataType = `${response.definition.success !== 'unknown' ? response.definition.success : ''}${response.definition.success !== 'unknown' && response.definition.errors !== 'unknown' ? ' | ' : ''}${response.definition.errors !== 'unknown' ? response.definition.errors : ''}`;
+  if (!responseDataType) responseDataType = 'unknown';
```

Comment:
It seems my changes were too complicated. This caused me to hit a bug 🥲 
I would like to simplify string generation and use immutable variables, so this is a refactoring suggestion.

```suggestion
  const responseDataSuccessType = response.definition.success !== 'unknown' ? response.definition.success : ''
  const responseDataTypeDelimiter = response.definition.success !== 'unknown' && response.definition.errors !== 'unknown' ? ' | ' : ''
  const responseDataErrorsType = response.definition.errors !== 'unknown' ? response.definition.errors : ''
  const responseDataType = responseDataSuccessType || responseDataErrorsType ? `${responseDataSuccessType}${responseDataTypeDelimiter}${responseDataErrorsType}` : 'unknown'
```